src/app/content/components/Toolbar.tsx

```diff
@@ -323,9 +389,15 @@ class Toolbar extends React.Component<{
 
 export default connect(
   (state: AppState) => ({
+    open: selectSearch.open(state),
     query: selectSearch.query(state),
+    results: selectSearch.results(state),
   }),
   (dispatch: Dispatch) => ({
+    onClose: () => {
+      dispatch(clearSearch());
+    },
     search: (query: string) => dispatch(requestSearch(query)),
+    toggleSearchSidebar: (open: boolean) => dispatch(toggleSearchSidebar(open)),
```

Comment:
not sure if you only put this here for testing, but we don't want this here. when you click the magnifying glass we want to `clearSearch`, the only way to close the sidebar without clearing it is to click on one of the result links

Reply:
We need the "openSearchResults" action for the "< Search Results" button that will open the results bar.

Reply:
right ok, but its not being used for that now